workaround for symbol error boost::system::system_category

/usr/bin/ccache  x86_64-pld-linux-g++  -fno-gcse -fno-omit-frame-pointer -ftemplate-depth-120 -Wall -Woverloaded-virtual -Wno-deprecated -Wno-strict-aliasing -Wno-write-strings -Wno-invalid-offsetof -fno-operator-names -Wno-error=array-bounds -Wno-error=switch -std=gnu++0x -Werror=format-security -Wno-unused-result -Wno-sign-compare -O2 -fwrapv -pipe -Wformat -Werror=format-security -gdwarf-4 -fno-debug-types-section -fvar-tracking-assignments -g2 -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -fPIC -march=x86-64 -gdwarf-4 -fno-debug-types-section -fvar-tracking-assignments -g2  -DNDEBUG    -Wl,--as-needed -Wl,--no-copy-dt-needed-entries -Wl,-z,relro -Wl,-z,combreloc CMakeFiles/gen-class-map.dir/gen-class-map.cpp.o CMakeFiles/gen-class-map.dir/idl.cpp.o  -o gen-class-map -rdynamic -lfolly -lglog ../../../../bin/libdouble-conversion.a -lpthread -ldl
/usr/bin/ld: CMakeFiles/gen-class-map.dir/gen-class-map.cpp.o: undefined reference to symbol 'boost::system::system_category()'
/usr/lib64/libboost_system.so.1.54.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [hphp/tools/bootstrap/gen-class-map] Error 1
make[2]: Leaving directory `/home/users/glen/rpm/BUILD/x86_64-linux/hiphop-php-HPHP-2.1.0/build'

--- hiphop-php-HPHP-2.1.0/hphp/tools/bootstrap/CMakeLists.txt	2013-10-06 01:35:42.071737427 +0300
+++ hiphop-php-HPHP-2.1.0/hphp/tools/bootstrap/CMakeLists.txt	2013-10-06 01:47:36.638238285 +0300
@@ -2,7 +2,7 @@
-target_link_libraries(gen-ext-hhvm folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} ${DL_LIB})
+target_link_libraries(gen-ext-hhvm folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} -lboost_system ${DL_LIB})
 
 add_executable(gen-infotabs "gen-infotabs.cpp" "idl.cpp")
-target_link_libraries(gen-infotabs folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} ${DL_LIB})
+target_link_libraries(gen-infotabs folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} -lboost_system ${DL_LIB})
 
 add_executable(gen-class-map "gen-class-map.cpp" "idl.cpp")
-target_link_libraries(gen-class-map folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} ${DL_LIB})
+target_link_libraries(gen-class-map folly ${LIBGLOG_LIBRARY} double-conversion ${LIBPTHREAD_LIBRARIES} -lboost_system ${DL_LIB})
